xcode/Subconscious/Shared/Services/Noosphere/SphereFS.swift

```diff
@@ -449,17 +463,23 @@ public final class SphereFS: SphereProtocol {
             ns_string_array_free(file_header_names)
         }
 
-        let name_count = file_header_names.len
-        guard var pointer = file_header_names.ptr else {
+        return file_header_names.toSwift()
+    }
+}
+
+extension slice_boxed_char_ptr_t {
+    func toSwift() -> [String] {
```

Comment:
This seems like a bad name but I struggled to find a better one.

Reply:
`.toStringArray()` ?

Reply:
Much better, thanks!